pkg/console/operator/sync_v400.go

```diff
@@ -365,9 +367,9 @@ func (co *consoleOperator) SyncConfigMap(
 		monitoringSharedConfig = &corev1.ConfigMap{}
 	}
 
-	telemeterClientIsAvailable, err := deploymentsub.IsTelemeterClientAvailable(co.monitoringDeploymentLister)
-	if err != nil {
-		return nil, false, "FailedTelemeterClientCheck", err
+	telemeterConfig, tcErr := co.GetTelemeterConfiguration(ctx, operatorConfig)
```

Comment:
nit, consistent naming
```suggestion
	telemetryConfig, tcErr := co.GetTelemeterConfiguration(ctx, operatorConfig)
```